plenum/server/replica.py

```diff
@@ -1912,7 +1923,8 @@ def order_3pc_key(self, key):
                           pp.ledgerId,
                           pp.stateRootHash,
                           pp.txnRootHash,
-                          pp.auditTxnRootHash if f.AUDIT_TXN_ROOT_HASH.nm in pp else None)
+                          pp.auditTxnRootHash if f.AUDIT_TXN_ROOT_HASH.nm in pp else None,
+                          self.node.get_primaries_for_view_no(pp.viewNo))
```

Comment:
After discussion and realizing that replicas can be removed, it's more safe to call `self.node.primaries` here

Reply:
fixed